### ISSUE TITLE
Analytics: Tracking activity for users between VA.gov and mobile app

### ISSUE THREAD

**Issue by ambikaroos** [department-of-veterans-affairs/va.gov-team#21689]
Do we have a way to answer questions such as these?

**Feature usage between desktop and mobile:**
- How many users check claims in both the app and VA.gov?
- How many users check appointments in both the app and VA.gov?
- How many users send messages in both the app and MHV?
[_etc, repeated for key features_]

**Overall frequency of usage between desktop and mobile:**
- For some subset of users, do they start to interact with any given feature more in the app than they do on VA.gov/MHV? (might yield insights on whether Vets have a clear preference of which channel to use to interact with VA)
- Are frequent mobile app users also frequent VA.gov users? (might yield insights on whether app is increasing overall engagement with VA from existing users)
- Are frequent or new mobile app users people who haven’t interacted much with VA.gov or MHV in the past? (might yield insights on whether app is allowing us to reaching new users)

**Comment by drjecker** [comment]
@ambikaroos Should we broaden the messaging question or add one to ask "How many users open messages in both the app and MHV?". I could see a user workflow in which they are using the app to keep tabs on messages (e.g. just opening) and then using MHV when they want to compose (because of drafts, because they are on MHV checking RX, being able to to use a keyboard, etc). So SM could be super useful to a user with out ever composing a message and how do we track that.

**Comment by ambikaroos** [comment]
DSVA Slack discussion on the topic here: https://dsva.slack.com/archives/C01KBMKPJ95/p1616181731001300

TLDR: This is something we will consider implementing post launch, when/before we expect significant traffic to the app. It would require collab cycle with VSP to implement on the VA.gov side as well as mobile app
